File: banda/AMarotoChivite-banda/AMarotoChivite-banda-Java/src/test/java/simulacionBandaMusica/factories/TeclistaFactoryTest.java
package simulacionBandaMusica.factories;

import org.junit.jupiter.api.Test;
import simulacionBandaMusica.models.Teclista;

import static org.junit.jupiter.api.Assertions.assertNotNull;

class TeclistaFactoryTest {

    @Test
    void createTeclistaaTest() {
        Teclista Teclista = TeclistaFactory.create();
        // Teclista no nulo, para ver si se han asignado correctamente los datos
        assertNotNull(Teclista);
    }
}

File: banda/AMarotoChivite-banda/AMarotoChivite-banda-Kotlin/src/test/kotlin/simulacionBandaMusica/factories/TeclistaFactoryTest.kt
package simulacionBandaMusica.factories

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test
import simulacionBandaMusica.models.Musico

internal class TeclistaFactoryTest {
    @Test
    fun createTeclistaaTest() {
        val Teclista: Musico.Teclista = TeclistaFactory.create()
        // Teclista no nulo, para ver si se han asignado correctamente los datos
        Assertions.assertNotNull(Teclista)
    }
}